DICOMPlugins/DICOMTID1500Plugin.py

```diff
@@ -6,22 +6,36 @@
 from collections import Counter
 
 import numpy
+import numpy as np 
 import random
-import pydicom
-
+# import pydicom
+from pydicom.sr.codedict import codes
 
 import slicer
 from DICOMLib import DICOMLoadable
 from base.DICOMPluginBase import DICOMPluginBase
 from SlicerDevelopmentToolboxUtils.mixins import ModuleLogicMixin
 
+slicer.util.pip_install("pydicom==2.4.4")
+import pydicom 
+slicer.util.pip_install("highdicom==0.22.0")
+
+try:
+    import highdicom as hd
+except ModuleNotFoundError:
+    #slicer.util.pip_install("highdicom==0.24.0")
+    slicer.util.pip_install("highdicom")
+    import highdicom as hd 
+print('highdicom version: ' + str(hd.__version__))
```

Comment:
It is preferable to use `logging` instead of `print` - you can do `logging.info` here.